### PR TITLE
test translation

### PR DESCRIPTION
**Pull request by chrisjsewell**

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 author = "the Executable Book Project"
 
 master_doc = "index"
+language = "fr"
```
Comment (chrisjsewell) [inline docs/conf.py:9]:
```suggestion
language = "ar"
```

**Comment by choldgraf** [comment]
I don't see any translations at all in the RTD builds 🤔 am I missing something?

**Comment by chrisjsewell** [comment]
yeh I'm not sure why that is, can you try locally

**Comment by choldgraf** [comment]
I'll try again when I'm at my computer

**Comment by chrisjsewell** [comment]
Its probably because I think you have to set up your project to be in the language in RTD (as well as in the conf.py?): https://docs.readthedocs.io/en/stable/localization.html#single-project-in-another-language

**Comment by choldgraf** [comment]
closing this one as I think this is resolved

**Pull request closed without merging**